test/components/advanced-options.test.tsx

```diff
@@ -1,13 +1,14 @@
 import React from "react";
-import { mount } from "enzyme";
+import { render, fireEvent, screen } from "@testing-library/react";
+import "@testing-library/jest-dom";
```

Comment:
If you put this side-effect-only import into `setupTests.js` then you wouldn't have to import it separately in each test file.

Reply:
Hm, this actually doesn't work for me. I've tried that before and it behaved strangely - sometimes one test would work, sometimes not. But when I ran `npm run test`, they consistently fail - there's an error about `toBeInTheDocument` not being defined.